examples/trainercontroller_configs/logging_controller_with_rank.yaml

```diff
@@ -16,8 +16,14 @@ operations:
     arguments:
       log_format: |
         Saving model in huggingface format at step: {trainer_state[global_step]}
-        Model saved in {args.output_dir}/checkpoint-{trainer_state[global_step]}
-      log_level: info
+        Model saved in {path}
+      log_level: warning
```

Comment:
Why not `info` level? the default fms-hf-level is to be raised to info so this can be a part of it.